mysql/init/01-databases.sql

```diff
@@ -0,0 +1,7 @@
+-- create regular and test databases
```

Comment:
Move this file to `config/` so we don't have to put `mysql` into git

Reply:
I can do that and put it under its own folder in `/config`, just because I don't want the database to load any other config files.